lib/redis.rb

```diff
@@ -3826,6 +3833,32 @@ def _normalize_move_wheres(where_source, where_destination)
 
     [where_source, where_destination]
   end
+
+  def _zsets_operation(cmd, args)
+    options = args.pop if args.last.is_a?(Hash)
```

Comment:
I think I'd prefer if you continued to use keyword arguments. To properly forward arguments from the other methods you can use `ruby2_keywords`.